snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -336,38 +329,38 @@ func (b *bootstrapper) Start(ctx context.Context, startReqID uint32) error {
 		return fmt.Errorf("failed to get linearization status: %w", err)
 	}
 	if linearized {
-		edge := b.Manager.Edge(ctx)
-		return b.ForceAccepted(ctx, edge)
+		return b.ForceAccepted(ctx, nil)
 	}
 
-	// If requested, assume the currently accepted state is what was linearized.
-	//
-	// Note: This is used to linearize networks that were created after the
-	// linearization occurred.
-	if b.Config.LinearizeOnStartup {
-		edge := b.Manager.Edge(ctx)
-		stopVertex, err := b.Manager.BuildStopVtx(ctx, edge)
-		if err != nil {
-			return fmt.Errorf("failed to create stop vertex: %w", err)
-		}
-		if err := stopVertex.Accept(ctx); err != nil {
-			return fmt.Errorf("failed to accept stop vertex: %w", err)
-		}
-
-		stopVertexID := stopVertex.ID()
-		b.Ctx.Log.Info("accepted stop vertex",
-			zap.Stringer("vtxID", stopVertexID),
+	// If a stop vertex is well known, accept that.
+	if b.Config.StopVertexID != ids.Empty {
+		b.Ctx.Log.Info("using well known stop vertex",
+			zap.Stringer("vtxID", b.Config.StopVertexID),
 		)
 
-		return b.ForceAccepted(ctx, []ids.ID{stopVertexID})
+		return b.ForceAccepted(ctx, []ids.ID{b.Config.StopVertexID})
 	}
 
-	if !b.StartupTracker.ShouldStart() {
-		return nil
+	// If a stop vertex isn't well known, treat the current state as the final
+	// DAG state.
```

Comment:
q: wouldn't this make stop vertex different for different nodes? Isn't this a problem that we linearize VMs in different nodes starting from different stop  vertexes?

Reply:
avalanchego doesn't support DAGs other than the Fuji X-chain and the Mainnet X-chain. This code is here so that the local network X-chain will be immediately linearized (it is assumed that the DAG is empty in this case).

Reply:
I'd add this comment to the codebase